osis-core/src/main/java/com/scality/osis/security/jwt/model/exception/AuthMethodNotSupportedException.java

```diff
@@ -1,5 +1,6 @@
 /**
  * Copyright 2020 VMware, Inc.
+ *Copyright 2022 Scality, Inc.
```

Comment:
here as well 😢

Reply:
Thank you fixed here: [`a09c90a` (#86)](https://github.com/scality/osis/pull/86/commits/a09c90ad697115275e9e1d7b2b5891560743a8a2)